packages/contracts/test/PoolMetadataRegistry.t.sol

```diff
@@ -63,30 +68,43 @@ contract PoolMetadataRegistryTest is IPoolMetadataRegistry, Test {
         assertFalse(isPool);
     }
 
-    function testIsPoolOwner() public {
+    function testIsPoolOwnerReturnsTrueWhenOwner() public {
         assertFalse(_poolMetadataRegistry.isPoolOwner(_basePool.getPoolId()));
+    }
 
-        vm.startPrank(address(15));
+    function testIsPoolOwnerFalseWhenNotOwner() public {
+        vm.startPrank(_poolOwner);
         assertTrue(_poolMetadataRegistry.isPoolOwner(_basePool.getPoolId()));
         vm.stopPrank();
     }
 
-    function testIfUpdatePoolMetadataRevert() public {
+    function testIfsetPoolMetadataRevertWhePoolNotRegistered() public {
         vm.expectRevert("Pool not registered");
 
         _poolMetadataRegistry.setPoolMetadata(0x0, _testMetadataCID);
     }
 
+    function testIfsetPoolMetadataRevertWhenNotOwner() public {
+        bytes32 notPoolId = _basePool.getPoolId();
```

Comment:
why is it called `notPoolId`?

Reply:
Just a misunderstanding, fixing it in the next commit